frontend/src/api/Problem.ts

```diff
@@ -38,6 +38,33 @@ export enum ProblemIOType {
   ArrayBoolean = 'ARRAY_BOOLEAN',
 }
 
+export const problemIOTypeToString = (key: ProblemIOType): string => {
+  switch (key) {
+    case ProblemIOType.String:
+      return 'String';
+    case ProblemIOType.Integer:
+      return 'Integer';
+    case ProblemIOType.Double:
+      return 'Double';
+    case ProblemIOType.Character:
+      return 'Character';
+    case ProblemIOType.Boolean:
+      return 'Boolean';
+    case ProblemIOType.ArrayString:
+      return 'ArrayString';
+    case ProblemIOType.ArrayInteger:
+      return 'ArrayInteger';
+    case ProblemIOType.ArrayDouble:
+      return 'ArrayDouble';
+    case ProblemIOType.ArrayCharacter:
+      return 'ArrayCharacter';
+    case ProblemIOType.ArrayBoolean:
+      return 'ArrayBoolean';
```

Comment:
Thoughts on if we have a custom map to display names if it would look better to do something like Double[]?

Reply:
I'd be slightly worried about these not exactly matching the style of other languages, since an identifier like `Double[]` seems specific to Java? I could be wrong on this, so I'll leave this open.

Reply:
I think most languages use brackets in some form but that does make sense - if we're going that route though, should we be concerned about languages like Python that use lists instead of arrays? e.g. Python lists would be analogous to Java Lists rather than arrays, which idk if that would affect our naming in any way

Reply:
Yeah that's true - what's your preference on this?

(Also if you're updating this right now, feel free to just change this and other nits if you want / are free)

Reply:
Idk I don't feel strongly either way (especially since this page isn't public facing yet) - so I'm open to leaving as is or changing it, whatever you think makes sense

Reply:
Ok, I think I'll leave this but happy to come back to it later.